index.html

```diff
@@ -110,6 +146,56 @@
   </p>
 </section>
 
+<!-- - - - - - - - - - - - - - -  Introduction  - - - - - - - - - - - - - - -->
+<section class="informative"> <h2>Introduction</h2>
+  <p>
+    In general, there are following groups of user scenarios for NFC:
```

Comment:
in general, you can group the user scenarios of NFC into the following groups:

Reply:
I was taught to use passive constructs in articles (and I assume specs) :).

Reply:
Maybe then 

>NFC user scenarios can be grouped as follows:

Reply:
BTW I just copied these use cases, so an English proofreading is due for the whole spec.